.github/workflows/build_aarch64.yml

```diff
@@ -0,0 +1,315 @@
+#
+# Copyright (c) 2014-present, The osquery authors
+#
+# This source code is licensed as defined by the LICENSE file found in the
+# root directory of this source tree.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR GPL-2.0-only)
+#
+
+# Due to a limitation in how GitHub Actions works, we can't reference
+# jobs in another file inside the `needs` statement.
+#
+# This configuration file takes care of Linux on aarch64.
+name: build_aarch64
+
+on:
+  push:
+    branches:
+      - 'master'
```

Comment:
Might be better not to scope this to master. We don't usually have other branches, but it would allow us to push something and test it with the arm64 CI before merge.

Reply:
Good point